src/command/delete.rs

```diff
@@ -0,0 +1,40 @@
+use atuin_common::api::LoginRequest;
+use eyre::Result;
+use structopt::StructOpt;
+
+use crate::command::or_user_input;
+use atuin_client::api_client;
+use atuin_client::settings::Settings;
+
+#[derive(StructOpt)]
+#[structopt(setting(structopt::clap::AppSettings::DeriveDisplayOrder))]
+pub struct Cmd {
+    #[structopt(long, short)]
+    pub username: Option<String>,
+
+    #[structopt(long, short)]
+    pub password: Option<String>,
+}
+
+impl Cmd {
+    pub async fn run(&self, settings: &Settings) -> Result<()> {
```

Comment:
Hmmm I see what you mean, though it could also be a little confusing. 

Perhaps literally forcing them to type "Delete my account" would be the best option 😂

Reply:
sure, can be both

Reply:
and perhaps add more SCREAMING WORDS in command about that it deletes account and data